shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -557,6 +557,27 @@ bool FlutterWindowsEngine::MarkExternalTextureFrameAvailable(
               engine_, texture_id) == kSuccess);
 }
 
+bool FlutterWindowsEngine::PostRasterThreadTask(
+    std::function<void()> callback) {
+  struct Captures {
+    std::function<void()> callback;
+  };
+  auto captures = std::make_unique<Captures>();
+  captures->callback = callback;
+  if (embedder_api_.PostRenderThreadTask(
+          engine_,
+          [](void* opaque) {
```

Comment:
Maybe I'm missing something, but can't you pass the callback using a lambda capture?

Reply:
Something like:

```
...
[&](void* opaque) {
   callback();
},
...
```

Reply:
`FlutterEngineProcTable`'s `PostRenderThreadTask` expects a C-style callback, i.e. a non-capturing lambda must be used here.

Reply:
Ah right, please ignore